package.json

```diff
@@ -4,13 +4,14 @@
   "description": "Make CSS easier and more maintainable by using JavaScript",
   "main": "dist/free-style.js",
   "typings": "dist/free-style.d.ts",
+  "jsnext:main": "dist.es2015/free-style.js",
   "files": [
     "dist/",
```

Comment:
Ahh, hard to do everything perfect. Are you working on this or would you like another PR?